test/parallel/test-global-console-exists.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const assert = require('assert');
+const events = require('events');
+
+const old_default = events.defaultMaxListeners;
+events.defaultMaxListeners = 1;
+
+const e = new events.EventEmitter();
+e.on('hello', function() {});
+
+assert.ok(!e._events['hello'].hasOwnProperty('warned'));
```

Comment:
Perhaps it would be better to use a public API to check, such as:

```javascript
assert.ok(!e.listeners('hello')[0].hasOwnProperty('warned'));
```